src/models/ratingcourse.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../db');
+
+const RatingCourse = {
+  async create(obj) {
+    const text = 'INSERT INTO ratingcourse VALUES($1, $2, $3)';
```

Comment:
Also can you add `RETURNING *` at the end of the query so that postgres returns the inserted rows. You can then use res.rows[0] to return the inserted line to the router.

Reply:
Corrected (normally)